app/controllers/answers_controller.rb

```diff
@@ -1,11 +1,37 @@
 class AnswersController < ApplicationController
 
-  def index
+  def new
+    @question = Question.find(params[:question_id])
+    @answer = Answer.new
   end
 
-  def show
+  def create
+    question = Question.find(params[:question_id])
+    answer = question.answers.new(content: params[:answer][:content], user_id: session[:user_id])
```

Comment:
seems like you should be able to say `question.answers.new(params[:answer])`.  The double dereference `[][]` seems like a smell.